includes/discovery/discovery-protocols.inc.php

```diff
@@ -215,8 +215,13 @@
                 continue;
             }
 
-            $local_ifName = $lldp['lldpNeighborPortId'][$IndexId][1];
```

Comment:
`$local_ifName` is used here https://github.com/librenms/librenms/blob/d5e6c52ae693a49cd494ee47b1c17abef815ab1b/includes/discovery/discovery-protocols.inc.php#L237 which is no longer set in both cases unless I'm misreading something?

Reply:
Looking at the rest of the code, it looks like $interface is expected as the 4th argument for discover_new_device(), so I've updated the code accordingly.